clientv3/naming/grpc.go

```diff
@@ -18,11 +18,10 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-
```

Comment:
@skyao can we mark this package as deprecated? Better, we should move it into the grpcproxy package as an internal implementation detail. 

There is no point to expose this anymore since gRPC already stop supporting it. When our users bump gRPC the same version as the etcd client, they cannot use this package anyway. If they use an older version of gRPC, they should also be using an older version of etcd client with naming support.

So we will lose nothing anyway.